spec/features/item_view_metadata_spec.rb

```diff
@@ -1,17 +1,20 @@
 require 'spec_helper'
 
 feature 'Item view metadata' do
-  before do
-    @current_user = double(
+  let(:current_user) do
```

Comment:
They use instance variables in the doc, though it may not be a best practice. 

http://www.relishapp.com/rspec/rspec-core/docs/hooks/before-and-after-hooks